lib/AST/Decl.cpp

```diff
@@ -5424,7 +5424,19 @@ void VarDecl::setSpecifier(Specifier specifier) {
                           StorageIsMutable_t(!isImmutableSpecifier(specifier)));
 }
 
-bool VarDecl::isAnonClosureParam() const {
+bool ParamDecl::isAnonClosureParam() const {
+  auto name = getName();
+  if (name.empty())
+    return false;
+
+  auto nameStr = name.str();
+  if (nameStr.empty())
+    return false;
+
+  return nameStr[0] == '$';
+}
+
+bool VarDecl::isWrapperOrAnonClosureParam() const {
```

Comment:
I think it makes sense to name this one simplify `isWrapper` since to be anonymous closure parameter object has to be a `ParamDecl`